docs/src/use/configure/configuration-files.md

```diff
@@ -495,3 +498,156 @@ npx eslint --config some-other-file.js **/*.js
 ```
 
 In this case, ESLint does not search for `eslint.config.js` and instead uses `some-other-file.js`.
+
+<a name="experimental-typescript-support">
+
+## ◆ Configuration Files written in TypeScript (new)
+
+</a>
+
+As of ESLint v10, you can write your ESLint configuration files in [TypeScript](https://www.typescriptlang.org).
+
+::: warning
+This feature is currently experimental and may change in future versions.
+:::
+
+To use this feature, you must have [`jiti`](https://www.npmjs.com/package/jiti) installed in your project.
```

Comment:
This link will be most updated one

```suggestion
To use this feature, you must have [`jiti`](https://jiti.unjs.io) installed in your project.
```

Reply:
why add a dependency on this service? the npm package is canonical, and links to the github repo.

Reply:
Stub for future docs website. Github link is also fine...

Reply:
i mean `unjs.io` - npm and github's websites are known to be stable, things that aren't probably should be minimized